recipes/premake/5.x/conandata.yml

```diff
@@ -1,7 +1,19 @@
 sources:
+  "5.0.0-beta2":
+    url: "https://github.com/premake/premake-core/archive/refs/tags/v5.0.0-beta2.tar.gz"
+    sha256: "84a7775fe0a90c0f649eb18b10779b84626ad8c58dea4a8f24cca83690dd47d4"
   "5.0.0-alpha15":
-    url: "https://github.com/premake/premake-core/releases/download/v5.0.0-alpha15/premake-5.0.0-alpha15-src.zip"
-    sha256: "880f56e7cb9f4945d1cb879f059189462c1b7bf62ef43ac7d25842dfb177dd53"
+    url: "https://github.com/premake/premake-core/archive/refs/tags/v5.0.0-alpha15.tar.gz"
```

Comment:
Please don't use the auto generated GitHub source archives when there are official downloads

Reply:
They are more or less identical, except the packaged source distributions include the "bootstrapped" build files for a dozen or so build configurations. These are no longer used by the updated PR and can be skipped for a smaller download.

Reply:
GitHub does generate the source archives on the fly, they are not necessary stable. They broke them once recently and broke dozen of projects. And in no way are they official from a project point of view. So please use the official download

Reply:
Eh, fine. Reverted.